app/components/review-point-list.jsx

```diff
@@ -0,0 +1,61 @@
+// https://github.com/EnCiv/civil-pursuit/issues/61
+
+'use strict'
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import ReviewPoint from './review-point.jsx'
+
+const ReviewPointList = ({ ReviewPoints = [], onDone = () => {} }) => {
+  const [rankedPoints, setRankedPoints] = useState(new Map())
```

Comment:
The rank of a point could already be set, if the user is returning to this step to review or change ranking.  The ReviewPoint has a property for the initial rank. 
```
  ReviewPoints=[{point, leftPoints, RightPoints, rank}]
```
Also, it's a typo on my part and this should be reviewPoints.  The leading capital is for react components, not regular properties.

This component needs to begin with that value, which could be undefined, or could be a most, least, or neurtral.

This component should go ahead and set reviewPoint.rank to shat the user give it. And if reviewPoint.rank does not exist it should be created.

This is mutating the prop passed, which is often advised against, but in this case it will be part of a shared object that several components are operating on.  But we need to be caution and may change the model at some pont. You were thinking of passing int in onDone and that's a valid model, but we'll see.